packages/ng/styles/definitions/select/_select-clearer.scss

```diff
@@ -7,6 +7,19 @@
 		display: inline-block;
 	}
 
+	:host-context(.textfield.mod-outlined) {
+		--palettes-50: var(--palettes-grey-50);
```

Comment:
Why redefining a generic palette here ? It seems odd.
If i understand correctly the goal we could jst remove the generic palette usage when it's not needed ?
`color: var(--palettes-700, var(--components-select-default-palette-700))`; -> `color: var(--components-select-default-palette-700);`